lib/b2c/sessions.ts

```diff
@@ -158,24 +575,30 @@ export class Sessions {
       options
     );
 
+    // The last_authenticated_at field from local authentication is still a string.
+    // Let's strongly type it now and then convert it to a Date.
+    type authFactorRow = Omit<AuthenticationFactor, "last_authenticated_at"> & {
```

Comment:
nit - I think the `Row` prefix comes from DB SQL munging but I don't know if it makes sense here. maybe `authFactorJSON`?